scripts/build_docker.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+bash scripts/gen_proto.sh
+if [ $? -ne 0 ]
+then
+  echo "gen proto failed"
+  exit 1
+fi
+
+sudo docker build --net=host . -f docker/Dockerfile -t  datascience-registry.cn-beijing.cr.aliyuncs.com/easyrec/easyrec:py36-tf1.15
```

Comment:
tag add easy_rec version is better. py36-tf1.15-0.4.8

Reply:
done